test/specs/percy.js

```diff
@@ -0,0 +1,38 @@
+import { describe } from '../utils/mochaw'
+import { supportedLanguages, fullTestCoverageLanguages } from '../percy.json'
+import { welcomeScenarios } from './scenarios/welcome'
+import { documentSelectorScenarios } from './scenarios/documentSelector'
+import { countrySelectorScenarios } from './scenarios/countrySelector'
+import { documentScenarios } from './scenarios/document'
+import { faceScenarios } from './scenarios/face'
+import { crossDeviceScenarios } from './scenarios/crossDevice'
+import { modalScenarios } from './scenarios/modal'
+import { navigationScenarios } from './scenarios/navigation'
+import { proofOfAddressScenarios } from './scenarios/proofOfAddress'
+import { hostAppHistoryScenarios } from './scenarios/hostAppHistory'
+import { accessibilityScenarios } from './scenarios/accessibility'
+import { userConsentScenarios } from './scenarios/userConsent'
+
+describe('Percy Visual Regression Tests', () => {
+  // Multiple language scenarios
+  fullTestCoverageLanguages.forEach((lang) => {
+    welcomeScenarios(lang)
+    documentSelectorScenarios(lang)
+    //countrySelectorScenarios(lang)
+    documentScenarios(lang)
+    faceScenarios(lang)
+    //crossDeviceScenarios(lang)
+    //modalScenarios(lang)
+    //navigationScenarios(lang)
+  })
+  // Note: The SDK works also with language tags that do not include region (e.g. 'en', 'es')
+  // We are passing the region here so we can fetch the right json file path (e.g. `en_US/en_US.json`).
+  supportedLanguages.forEach((lang) => {
+    //welcomeScenarios(lang)
+  })
+  // PoA is only available in en
+  proofOfAddressScenarios()
+  //accessibilityScenarios()
+  //hostAppHistoryScenarios()
+  //userConsentScenarios()
```

Comment:
Why are some scenarios commented out? If we are not planning to test those now we can just delete them and bring them back when needed

Reply:
Oh, this was my mistake, I thought I had commented them out :D